src/randoop/main/Help.java

```diff
@@ -5,13 +5,13 @@
 import java.util.Arrays;
 import java.util.List;
 
+import randoop.Globals;
```

Comment:
I noticed this earlier.  Apparently, I had put the order in the project settings and those are not in the repo.  I changed my global settings so that other packages are first, then plume, then randoop